pkg/kv/kvtest/store.go

```diff
@@ -0,0 +1,319 @@
+package kvtest
+
+import (
+	"bytes"
+	"context"
+	"errors"
+	"fmt"
+	"testing"
+
+	"github.com/go-test/deep"
+	nanoid "github.com/matoous/go-nanoid/v2"
+	"github.com/treeverse/lakefs/pkg/kv"
+)
+
+type makeStore func(t *testing.T, ctx context.Context) kv.Store
+
+var runTestID = nanoid.MustGenerate("abcdef1234567890", 8)
```

Comment:
Nit: Why bother to limit it to digits?

Reply:
just to have something easy to read and add couple of more letters ;)